### PR TITLE
To white label dashboard, make it possible to override dashboard font…

### PR DESCRIPTION
**Pull request by panditrashmi**
…s and colors (https://github.com/eclipse/che/issues/8421)

kind/enhancement
### What does this PR do?
This change adds assets/branding/branding.css into index.html after application CSS and makes it possible to while label dashboard. branding.css is added after application.css so that custom dashboard assembly man override styles as nneeded.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/8421

###
Signed-Off by
rashmi.pandit@softwareag.com

### CODE REVIEW

File: dashboard/src/assets/branding/branding.css [hunk 1]
```diff
@@ -0,0 +1 @@
+/** Add your branding customizations here */
```
Comment (benoitf) [inline dashboard/src/assets/branding/branding.css]: hello, FYI there is a missing EOF line

**Comment by benoitf** [comment]
Hello @panditrashmi 
In order to accept contribution, you need to sign the Eclipse ECA and sign your commit with Signed-Off
Ip-Validation should be green to accept the Pull Request
https://dev.eclipse.org/eclipse-webhook/services/status_details.php?id=5ad50ed57ee7f

Thanks

**Comment by benoitf** [comment]
ci-build

**Comment by panditrashmi** [comment]
Hi,
I have signed ECA (with rashmi.pandit@softwareag.com). Not sure why ip-validation failed. Somehow it has email rasp@softwareag.com instead of rashmi.pandit@softwareag .com. Do I need to create a new pull request?
Thanks,
Rashmi

**Comment by benoitf** [comment]
@panditrashmi you should squash all your commits and set the author email to be  rashmi.pandit@softwareag.com and append Signed-Off by with this email as well

**Comment by panditrashmi** [comment]
Thank you! Took care of it. Signed-Off by rashmi.pandit@softwareag.com

**Comment by benoitf** [comment]
@panditrashmi you still have to squash your 6 commits into only one to ensure SignedOff is on all commits (or amend each commit)

**Pull request closed without merging**